plug-1.0.0-rc2/docker-compose.yml

```diff
@@ -12,15 +12,15 @@ services:
   #     block-network:
   #       ipv4_address: 10.5.0.20        
   
-  tx-node-1:
-    container_name: tx-node-1
+  tx-app-node:
```

Comment:
The app runs on the NodeJs framework, but it isn't a node like a plug-node.
I suggest we just call this tx-app and leave out node to avoid confusion.